cpu/mips_pic32_common/periph/uart.c

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2016, Imagination Technologies Limited and/or its
```

Comment:
Here and in all other files: please use "Copyright (C) 2016 ...", for consistency.